build/sync/plan.yml

```diff
@@ -0,0 +1,30 @@
+checks:
+  - type: repo_is_clean
+    params:
+      repo: template
+  - type: repo_is_clean
+    params:
+      repo: plugin
+actions:
+  - paths:
+      - build
+      - webapp/tests
+    actions:
+      - type: overwrite_directory
+        params:
+          create: true
+  - paths:
```

Comment:
As mentioned in https://github.com/mattermost/mattermost-plugin-agenda/pull/53#discussion_r468395603 it seams weird to see a `diff` between the file here and in the agenda repo. Where does it come from?

Reply:
The `plan.yml` file committed by me to the sync PRs have been "what was used for that plugin to sync with the starter template". If a file was already sync'd, I didn't include it in the `plan.yml`. The goal here was to document when needed to be sync'd and to show how it was done. That hasn't proven to be that useful other than for my own reference to sync other repos.

The goal should instead be to, as we've discussed, have one source-of-truth `plan.yml` to sync each repo.